CMakeLists.txt

```diff
@@ -232,9 +231,7 @@ else()
 endif()
 
 
-if(DETECT_PKG_CONFIG)
-  find_package(PkgConfig)
-endif()
+find_package(PkgConfig)
```

Comment:
@rcurtin I still understand why that `pkg_check_modules` call is there. The HDF5 libs currently don't provide `.pc` files (I'm trying to get `.pc` support integrated into upstream HDF5).